### PR TITLE
Add nethermind-node to linea-mainnet and linea-sepolia

### PR DESCRIPTION
**Pull request by stdevMac**
Add Support for Nethermind on `linea-mainnet` and `linea-sepolia`.

### CODE REVIEW

File: docker/linea-mainnet/docker-compose.yml [hunk 1]
```diff
@@ -69,3 +69,20 @@ services:
       - ./besu-genesis.json:/var/lib/besu/genesis.json:ro
       - ../config/linea-besu-sequencer/log4j.xml:/var/lib/besu/log4j.xml
       - ./static-nodes.json:/var/lib/besu/static-nodes.json
+
+  nethermind-node:
+    image: nethermindeth/nethermind:master
```
Comment (julien-marchand) [inline docker/linea-mainnet/docker-compose.yml]:
```suggestion
    image: nethermindeth/nethermind:1.29.1
```

File: docker/linea-sepolia/docker-compose.yml [hunk 2]
```diff
@@ -71,3 +71,20 @@ services:
       - ./besu-genesis.json:/var/lib/besu/genesis.json:ro
       - ../config/linea-besu-sequencer/log4j.xml:/var/lib/besu/log4j.xml
       - ./static-nodes.json:/var/lib/besu/static-nodes.json
+
+  nethermind-node:
+    image: nethermindeth/nethermind:master
```
Comment (julien-marchand) [inline docker/linea-sepolia/docker-compose.yml]:
```suggestion
    image: nethermindeth/nethermind:1.29.1
```

File: docker/linea-mainnet/docker-compose.yml (hunk 1 above)
Comment (julien-marchand) [reply docker/linea-mainnet/docker-compose.yml]: We prefer to stick to fixed images to be intentional when updating nodes.

File: docker/linea-mainnet/docker-compose.yml [hunk 3]
```diff
@@ -69,3 +69,20 @@ services:
       - ./besu-genesis.json:/var/lib/besu/genesis.json:ro
       - ../config/linea-besu-sequencer/log4j.xml:/var/lib/besu/log4j.xml
       - ./static-nodes.json:/var/lib/besu/static-nodes.json
+
+  nethermind-node:
```
Comment (julien-marchand) [inline docker/linea-mainnet/docker-compose.yml:73]:
```suggestion
  nethermind-node:
      hostname: nethermind-node
      container_name: nethermind-node
```

File: docker/linea-sepolia/docker-compose.yml [hunk 4]
```diff
@@ -71,3 +71,20 @@ services:
       - ./besu-genesis.json:/var/lib/besu/genesis.json:ro
       - ../config/linea-besu-sequencer/log4j.xml:/var/lib/besu/log4j.xml
       - ./static-nodes.json:/var/lib/besu/static-nodes.json
+
+  nethermind-node:
```
Comment (julien-marchand) [inline docker/linea-sepolia/docker-compose.yml:75]:
```suggestion
  nethermind-node:
      hostname: nethermind-node
      container_name: nethermind-node
```

File: docker/linea-mainnet/docker-compose.yml (hunk 3 above)
Comment (julien-marchand) [reply docker/linea-mainnet/docker-compose.yml:73]: Aligning convention with other services

**Comment by julien-marchand** [comment]
Thanks for this contribution! I just made some minor comments.

File: docker/linea-mainnet/docker-compose.yml (hunk 1 above)
Comment (stdevMac) [reply docker/linea-mainnet/docker-compose.yml]: This is while we don't have it as a stable release. That's when I will change it from draft to ready and update it with the expected tag. Thanks for pointing out anyway!

File: docker/linea-sepolia/docker-compose.yml (hunk 2 above)
Comment (stdevMac) [reply docker/linea-sepolia/docker-compose.yml]: Same

File: docker/linea-mainnet/docker-compose.yml (hunk 3 above)
Comment (stdevMac) [reply docker/linea-mainnet/docker-compose.yml:73]: Will do it soon!

File: docker/linea-sepolia/docker-compose.yml (hunk 4 above)
Comment (stdevMac) [reply docker/linea-sepolia/docker-compose.yml:75]: Will do it soon!

**Comment by julien-marchand** [comment]
Hey @stdevMac we'll put a bit more emphasis on that part of the codebase (basically redirecting the docs site to that place), it would be great to have this merge. Any blocker?

**Comment by stdevMac** [comment]
Hey @julien-marchand! 

This PR is ready! Each time we release a new version, I will push an update in this repo with the new image!

**Comment by Othryades** [comment]
@stdevMac Thank you for the new release with Linea network support! I have been testing it, and I noticed a odd behavior with the **eth_syncing** RPC method while syncing a Nethermind node on Linea mainnet:

- During synchronization, the **eth_syncing** response consistently shows currentBlock: 0x0, even though the logs indicate that the node is syncing correctly.
- Once the sync is complete, eth_syncing correctly returns false.
- **Expected behavior**: currentBlock in the eth_syncing response should increment towards highestBlock during synchronization.

Setup:
	•	Nethermind version: 1.30.0-rc docker img
	•	Network: Linea mainnet
	•	Configuration: Using the default linea-mainnet from the docker-cpomsoe.yaml.

Let me know if I can provide any additional details or logs.

**Review by julien-marchand — APPROVED** [review]

**Review by Othryades — APPROVED** [review]

**Review by thedarkjester — APPROVED** [review]

**Pull request merged**